packages/memcpy/memcpy.0.2.0/opam

```diff
@@ -0,0 +1,18 @@
+opam-version: "1.2"
+maintainer: "yallop@gmail.com"
+author: "yallop@gmail.com"
+homepage: "https://github.com/yallop/ocaml-memcpy"
+dev-repo: "http://github.com/yallop/ocaml-memcpy.git"
+bug-reports: "http://github.com/yallop/ocaml-memcpy/issues"
+license: "MIT"
+build: [[make]]
+install: [[make "install"]]
+build-test: [[make "test"]]
```

Comment:
if you tag these `org:mirage` I can also add to docs.mirage.io for central docs easily